LayoutTests/platform/mac-monterey/css1/box_properties/acid_test-expected.txt

```diff
@@ -0,0 +1,68 @@
+layer at (0,0) size 800x600
```

Comment:
How does this end up touching `mac-monterey`? Oh, is this oddness from the fact that `mac-ventura` already existed, given `mac-monterey` goes from inheriting from `mac` to inheriting from `mac-ventura`?

Reply:
I believe that's how this inheritance chain works.